### PR TITLE
Chore/PageSection divider unification

### PR DESCRIPTION
**Pull request by ChefHutch**
Preview: https://new-page-sections.netlify.app

- Delete old divider SVG and replace with new path approach
- Implement new path approach across app where PageSection is used
- Refactor and rename props

### CODE REVIEW

**Review by Chef-Cheems — APPROVED** [review]

File: src/views/Lottery/index.tsx [hunk 1]
```diff
@@ -42,15 +42,15 @@ const Lottery = () => {
...
+      <PageSection background={TITLE_BG} index={1} hasCurvedDivider={false}>
         <Hero />
       </PageSection>
       <PageSection
         containerProps={{ style: { marginTop: '-30px' } }}
         background={GET_TICKETS_BG}
         concaveDivider
-        concaveBackgroundLight="#7645D9"
-        curvePosition="top"
+        clipFill={{ light: '#7645D9' }}
```
Comment (hachiojidev) [inline src/views/Lottery/index.tsx:52]: If I am reading this correctly `clipFill` is supposed to be a `string`.

Comment (hachiojidev) [reply src/views/Lottery/index.tsx:52]: Looking at `PageSection/types.ts`

Comment (ChefHutch) [reply src/views/Lottery/index.tsx:52]: I think it's all good hachi? It's an object with two keys - light & dark, both of which are strings? 

```
{
  light: string
  dark?: string
}
```

You might have spotted something I've missed though!

Comment (hachiojidev) [reply src/views/Lottery/index.tsx:52]: Ah yeah you're right. My bad.

**Review by hachiojidev — APPROVED** [review]

**Pull request merged**